spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/ml/MLPipelineEventProcessor.scala

```diff
@@ -18,27 +18,25 @@
 package com.hortonworks.spark.atlas.ml
 
 import scala.collection.mutable
-
```

Comment:
Revert this change, should have a blank line.

Reply:
Thanks Jerry's review. 
1. For the global cache, this would not be a big issue since we only track ml related pipeline information. one spark application would have more than 1k pipeline related action. 

2. For the sql part, if uses do not have the ml job, the sql would run independently.